standards/MEETUP_TALKS.md

```diff
@@ -0,0 +1,69 @@
+
+- 4 weeks to go
+  - Create a [new issue][link_demo_issue] in the [talks][link_talks_repo] repo. Placing your events in this repo, allows other meetup organisers to see which speakers are active. They may be able to ask them to do a talk at their group.
+  - Confirm speakers and get the following from each of them:
+    - Speaker's bio
+    - Title of talk
+    - Synopsis of talk
+    - Duration of talk - Generally talks fall under the following category based on their duration:
+      - 5-15m lightning
+      - 25m-45m regular talks + 5-10m Q&A
+      - &gt; 60m long talks - it's worth advising the speaker to avoid talks of this duration unless some form of continual Q&A or audience participation takes place.
+  - Share this with venue if relevant. Our hosts Skillsmatter have their own website and newsletter which provides free advertisement to their event. It's often helpful to gain a similar relationship in your meetup as the host will often have a bigger audience reach.
+  - Add to meetup/group event page, enable RSVP if you have the capability.
+- 3 weeks
+  - Send out reminders via meetup, twitter or other social media.
+- 2 weeks
+  - Add lightning speakers
+  - Send out reminders via meetup, twitter or other social media.
+- 1 week to go
+  - Send a follow up email with speakers:
+    - Send details of location to venue, suggested route from nearest bus/train station.
+  - Send out reminders via meetup, twitter or other social media
+- On the day
+  - Decide running order of speakers
+
+## Time keeping
+
+If you have more than one speaker, time keeping is essential. Failing to do so could potential mean other speakers lose their opportunity to speak because your venue has limited hours allows meetups between the hours of 6pm-10pm.
+
+Ensure that all speakers are aware of this limitation.
+
+The recommended approach advise the speaker of time left, is to have warnings at 5min intervals starting at 15mins remaining of talk duration. Tip: you can tell the speaker through the use of cards with the time remaining or verbally. The latter is useful to remember since you won't have a mic so the video recording if you have one will not pick up the prompts.
+
+Also don't forget to time keep the Q&A section too!
+
+## Hints and tips
+
+- **Automate**: Simplify your workflow for organising meetups by making use of [issue templates][link_gh_templates]. Templates can generate a standardised checklist. The use of [query parameters][link_gh_query_params] can also add labels and a title. The London template can be found [here][link_lon_talks_template].
+- **Finding a venue**: Often companies that use Rust will provide a venue for free. There's a list of companies on the [Rust front page][link_rust_friends], some of these companies might be near to your city.
+- **Booking venue**: it's often easier to pre-book in quarters i.e. January to March, April to June, etc.
+- **Beg, steal, borrow**: if you attend other techical meetups, try booking their venue.
```

Comment:
Can we change the wording here? I don't think "beg" and "steal" are the right words here.